ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/sflow-interfaces.j2

```diff
@@ -0,0 +1,36 @@
+{# SFlow Interfaces #}
+{% set port_channel_interfaces_sflow = [] %}
+{% set ethernet_interfaces_sflow = [] %}
+{% for port_channel_interface in port_channel_interfaces | arista.avd.natural_sort %}
+{%     if port_channel_interfaces[port_channel_interface].sflow.enabled is arista.avd.defined or
+           port_channel_interfaces[port_channel_interface].sflow.egress.enabled is arista.avd.defined or
+           port_channel_interfaces[port_channel_interface].sflow.egress.unmodified_enabled is arista.avd.defined %}
+{%         do port_channel_interfaces_sflow.append(port_channel_interface) %}
+{%     endif %}
+{% endfor %}
+{% for ethernet_interface in ethernet_interfaces | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
+{%     if ethernet_interface.sflow.enabled is arista.avd.defined or
+           ethernet_interface.sflow.egress.enabled is arista.avd.defined or
+           ethernet_interface.sflow.egress.unmodified_enabled is arista.avd.defined %}
+{%         do ethernet_interfaces_sflow.append(ethernet_interface) %}
+{%     endif %}
+{% endfor %}
+{% if port_channel_interfaces_sflow | length > 0 or ethernet_interfaces_sflow | length > 0 %}
+
+#### SFlow Interfaces
+
+| Interface | Ingress | Egress | Egress Unmodified |
```

Comment:
Please collapse the `Egress and Egress Unmodified` into a single column.
The value can then be `True` or `True (unmodified)`

Also since the values are `True`/`False` I think we should call the columns `Ingress Enabled` and `Egress Enabled`